### ISSUE TITLE
Errors in Mozilla Firefox with mouse tracking enabled

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3654]
**Reported by k_kolev1985 on 2013-11-20 10:42**
If I try to read text in objects (no matter if in the interface or in the virtual document itself) in Mozilla Firefox, by using the "mouse tracking" feature of NVDA, the screen reader may give errors (sounds).

Steps to reproduce it:
1. Enable the mouse tracking of NVDA, and also, just to be sure, enable the option to report the object type under the mouse cursor.
2. Open Firefox and load a page in it.
3. Try reading the page with the mouse, hovering it over the text on the page. Error sounds from NVDA may be heard, but not always.
4. Activate a link, witch points to a file for download (the file type doesn't matter).
5. Try again to read the text in the objects in the "File download" window. NVDA will most probably play its error sound again.

In both cases, I think NVDA reads the text under the mouse cursor, but also plays its error sound.

My configuration:
- OS: Windows 7 Ultimate SP1, 32-bit, in bulgarian (with locale settings also set to bulgarian);
- NVDA version: snapshot next-9985,6740b92, in bulgarian;
- Firefox version: 25.0.1, in bulgarian;

Interestingly enough, the error does not occur with a clean (with no add-ons installed) portable copy of NVDA 2013.3RC2. I think an add-on may be the culprit, but I can't figure out witch one, unfortunately. I'm attaching a NVDA log file with logging level set to "input/output". Hope it helps.
**Blocking #3837**

**Comment by feerrenrut** [comment]
One such error is:
```
ERROR - eventHandler.executeEvent (10:31:07.707):
error executing event: mouseMove on <NVDAObjects.IAccessible.mozilla.Mozilla object at 0x05160EF0> with extra args of {'y': 401, 'x': 1514}
Traceback (most recent call last):
  File "eventHandler.pyc", line 152, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\__init__.pyc", line 930, in event_mouseMove
  File "NVDAObjects\IAccessible\__init__.pyc", line 142, in expand
AttributeError: 'NoneType' object has no attribute 'rindex'
```
This is considered fairly harmless, we should stop it from occurring so that it is no longer a red herring.

**Comment by Adriani90** [comment]
I am on last Alpha and Firefox 63.0.3. There are no errors when tracking the mouse anymore. @k-kolev1985, @feerrenrut can you still reproduce it?

**Comment by leonardder** [comment]
AH, I did not know this issue existed.
Fixed in #9029
